manifests/common/plumgrid.pp

```diff
@@ -0,0 +1,16 @@
+# Common class used by Plumgrid Neutron Plugin.
+
+class openstack::common::plumgrid {
+  # forward all ipv4 traffic
+  # this is required for the vms to pass through the gateways public interface
+  sysctl::value { 'net.ipv4.ip_forward': value => '1' }
```

Comment:
Router profile(profile/neutron/router.pp) should be excluded for PLUMgrid deployments, it's not needed.